dali/python/nvidia/dali/fn.py

```diff
@@ -76,6 +76,7 @@ def op_wrapper(*inputs, **arguments):
         return op_class(**init_args)(*inputs, **call_args)
 
     op_wrapper.__name__ = wrapper_name
+    op_wrapper.__qualname__ = wrapper_name
```

Comment:
What is it?

Reply:
LMGIFY ;)
https://stackoverflow.com/questions/58108488/what-is-qualname-in-python

Reply:
OK, thanks for explanation.